src/main/scala/org/dbpedia/spotlight/wikistats/utils/SpotlightUtils.scala

```diff
@@ -49,6 +49,68 @@ object SpotlightUtils extends Serializable{
     Source.fromFile(stopWordLoc).getLines().toSet
   }
 
+  /*
+ Logic to create the Memory Token Store
+  Input:  - List of all Token types
+  Output: - Memory Store with the Token information
+ */
+  def createTokenTypeStore(tokenTypes: List[TokenType]): MemoryTokenTypeStore =  {
+
+    val tokenTypeStore = new MemoryTokenTypeStore()
+    val tokens = new Array[String](tokenTypes.size + 1)
+    val counts = new Array[Int](tokenTypes.size + 1)
+
+    println ("Naveen in tokentypes:")
+
+    tokenTypes.map(token => {
+      tokens(token.id) = token.tokenType
+      counts(token.id) = token.count
+
+    })
+
+    tokenTypeStore.tokenForId  = tokens.array
+    tokenTypeStore.counts = counts.array
+    tokenTypeStore.loaded()
+
+    tokenTypeStore
+  }
+
+  /*
+      Logic to concatnate two strings with the space. This function is used the reduceByKey operation
+      to contact two paragraphs.
+      Input:  - Two Strings to concatenate
+      Output: - Concatenated String with Space
+  */
+
+
+  def stringConcat (str1: String, str2: String): String = {
+
+    str1 + " " + str2
```

Comment:
yeah, i think this will create 3 strings. I can change to stringbuilder. 
The reason i am doing this is suppose you have two paragraphs,
1. This is paragraph one.
2. This is paragraph two.

The reduceByKey with string concat will return     "This is paragraph one.This is paragraph two."

So, here "one.This" is considered as one word which is not.

Hence I am adding a space during concatenation.
I am gonna change this to StringBuilder, I think that is efficient compared to String